### PR TITLE
Added an option to switch between field and getter return types.

### PR DESCRIPTION
**Pull request by DaSilva2010**
We use Optional (guava) in our getters for fields that are nullable. But we don't want Optional for the generated classes. So this is our suggestion to make the generated return type more user-defined.

### CODE REVIEW

**Comment by Shredder121** [comment]
Thanks for the PR, may I ask if you could maybe remove the whitespace changes?
I'll also add some more comments on a few lines.

File: querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractExporterMojo.java [hunk 1]
```diff
@@ -77,6 +77,13 @@
     private boolean handleMethods = true;
 
     /**
+     * switch for usage of field types instead of getter types
+     *
+     * @parameter default-value-false
```
Comment (Shredder121) [inline querydsl-maven-plugin/src/main/java/com/querydsl/maven/AbstractExporterMojo.java]: This should be `default-value=false`.

**Comment by DaSilva2010** [comment]
I don't understand why the CI build fails since there were only whitespace changes between this and the last successful build.

**Comment by timowest** [comment]
Looks good.

**Comment by Shredder121** [comment]
Yes, looks good.
Thanks again for the PR!

**Pull request merged**